application/src/main/resources/application.properties

```diff
@@ -8,12 +8,19 @@ logging.level.org.springframework.web=INFO
 logging.level.uk.gov.hmcts.ccd=INFO
 logging.level.org.elasticsearch.client=INFO
 
-spring.datasource.url=jdbc:postgresql://${DEFINITION_STORE_DB_HOST:localhost}:${DEFINITION_STORE_DB_PORT:5000}/${DEFINITION_STORE_DB_NAME:ccd_definition}${DEFINITION_STORE_DB_OPTIONS:}
-spring.datasource.username=${DEFINITION_STORE_DB_USERNAME:ccd}
-spring.datasource.password=${DEFINITION_STORE_DB_PASSWORD:password}
-spring.datasource.type=com.zaxxer.hikari.HikariDataSource
-spring.datasource.hikari.maximum-pool-size=${DEFINITION_STORE_DB_MAX_POOL_SIZE:5}
-spring.datasource.hikari.connection-timeout=60000
+write.datasource.url=jdbc:postgresql://${DEFINITION_STORE_DB_HOST:localhost}:${DEFINITION_STORE_DB_PORT:5000}/${DEFINITION_STORE_DB_NAME:ccd_definition}\
+  {DEFINITION_STORE_DB_OPTIONS:}&targetServerType=master
+write.datasource.username=${DEFINITION_STORE_DB_USERNAME:ccd}
```

Comment:
It could be better to follow a semantic hierarchy of properties, starting with app group and then followed by the app name and then etc etc . For example:
ccd.data-store-api.data-source.write-db.x
ccd.data-store-api.data-source.write-db.y.z

Reply:
This comment may apply to similar other application parameters